### ISSUE TITLE
look not running

### ISSUE THREAD

**Issue by zbenafia** [looker/lookr#22]
Hi, I'm getting the following error when I run `run_look(114)`

```
Error in .jcall("RJavaTools", "Ljava/lang/Object;", "invokeMethod", cl,  : 
  io.swagger.client.ApiException: {"message":"Not found","documentation_url":"http://docs.looker.com/"}
```
anyone knows the meaning of this?

**Comment by maxcorbin** [comment]
@zbenafia The "Not Found" error is an HTTP 404 status. This indicates that:

1. The API user does not have permissions for this endpoint (the user does not have the `see_looks` or `access_data` permissions)
2. The endpoint does not exist (unlikely unless you have modified the Java classes or the R code used to call those classes)
3. The resource you are trying to retrieve does not exist (confirm that Look 114 is a real look that exists and that the user has access to it via content access controls)

tl;dr Your best next step is to sudo as the API user and navigate to the Look to confirm you can run the Look in the front end. It is likely a permissions or content access error

**Comment by samudzi** [comment]
I've been running into this same issue.  I'm using the API key of an admin user who not only has full access to the entire instance, but is also the creator of the Look I'm trying to pull

**Comment by Nigel-Brown** [comment]
Using the following code: `data <- sdk$userSession$lookApi$look(look_id = 27185, config = sdk$oauthHeader)$content`
I'm receiving the following error message: `Error in self$fromJSONObject(QueryObject) : 
  object 'TODO_OBJECT_MAPPING' not found`.
 If I leave the config argument out of the function call, the response is a 401 error
